src/utility.jl

```diff
@@ -0,0 +1,30 @@
+"""
+```
+visualize_optical_flow_hsv(flow)
+```
+
+Return the visualization of an optical flow vector `flow` in hsv color space.
```

Comment:
It would be great if we could follow the documentation convention that I have started using in this package: https://zygmuntszpak.github.io/ImageContrastAdjustment.jl/dev/

For example:
```julia
visualize_flow(ColorBased(), flow)
```
Returns an image that matches the dimensions of the input matrix and that depicts the orientation and magnitude of the optical flow vectors using the HSV color space. 

# Details
Hue encodes angle between optical flow vector and the x-axis in the  image plane; 
saturation encodes the ratio between the individual vector magnitudes and the maximum magnitude 
among the whole motion field, and value always equals one. 

# Arguments
The `flow` parameter needs to be a two-dimensional arrays of length-2 vectors (of type `SVector`) which represent the displacement of each pixel. 
(*It is worth mentioning something about the row,column vs x,y convention. Perhaps the function should have a keyword argument that indicates what convention the flow vector are using. The default should be row,column and that's why we change to x,y inside the function. However, we should let the user toggle this*)

# Example
```julia
# Insert a minimum working example here...
```

# Reference
 Let's track down the article that originally proposed or popularised the HSV visualization of the optical flow.

Reply:
@zygmuntszpak I am not able to find the exact article proposing the HSV visualization. I could find only [this](http://vision.middlebury.edu/flow/flowEval-iccv07.pdf) and [this](https://people.csail.mit.edu/celiu/OpticalFlow/), where they are kind of using the visualization. 
I also went through [this](https://arxiv.org/pdf/1612.02590.pdf) survey paper but couln't find the source any where. Do you know the exact origin? 
Thank you

Reply:
Hi @Deepank308 , 
The reference that is often given is:

 S. Baker,  D. Scharstein,  JP Lewis,  S. Roth,  M.J. Black,  and R. Szeliski.  A database and
evaluation methodology for optical flow.International Journal of Computer Vision, 92(1):1–31, 2011

However, after skimming through the paper I don't actually see the definition of it there either! 
I found this which "kinda" mentions it but also doesn't specify it precisely:
https://hciweb.iwr.uni-heidelberg.de/sites/default/files/node/files/1689085625/Visualization.pdf

Reply:
@zygmuntszpak , even in the link you provided, the "S. Baker ..... " is referenced.
So, I am also adding it as a reference here.